src/main/java/baseball/ResultTell.java

```diff
@@ -0,0 +1,27 @@
+package baseball;
+
+import java.util.List;
+
+public class ResultTell {
+
+    public static String HowResult(int a, int num, List computerList, List playerList) {
```

Comment:
저번에 못보고 지나갔는데 List 안에 무엇이 있는지 적어줘야 합니다.
```suggestion
    public static String HowResult(int a, int num, List<Integer> computerList, List<Integer> playerList) {
```
다른 곳도 마찬가지로 바꿔주세요